appveyor.yml

```
@@ -46,19 +44,16 @@ build_script:
   - cmd: cd %APPVEYOR_BUILD_FOLDER%
   - cmd: md build
   - cmd: cd build
-  # detect revision
-  - cmd: python %APPVEYOR_BUILD_FOLDER%\utils\detect_revision.py > %APPVEYOR_BUILD_FOLDER%\WL_RELEASE
-  - cmd: set /p WIDELANDS_VERSION_STRING=<%APPVEYOR_BUILD_FOLDER%\WL_RELEASE
+  # detect version
+  - cmd: set /p WIDELANDS_NEXT_VERSION=<%APPVEYOR_BUILD_FOLDER%\NEXT_STABLE_VERSION
+  - cmd: set SHORT_ID=%APPVEYOR_REPO_COMMIT:~0,7%
```

You could set `WIDELANDS_VERSION_STRING` here, or even the full setup name to remove the duplication below
```suggestion
  - cmd: set SHORT_ID=%APPVEYOR_REPO_COMMIT:~0,7%
  - cmd: set WIDELANDS_SETUP_NAME=Widelands-%CONFIGURATION%-%PLATFORM%-%WIDELANDS_NEXT_VERSION%~git(%SHORT_ID%@%APPVEYOR_REPO_BRANCH%)
```